.github/workflows/build-deploy.yaml

```diff
@@ -12,66 +12,4 @@ jobs:
         uses: actions/checkout@v2
       - name: Run Tests
         run: |
-          make ci        
-  build-docker-image:
```

Comment:
Since it's not building or deploying it should be renamed to something indicated that it runs ci or tests.